### ISSUE TITLE
Phase out odoo for our website

### ISSUE THREAD

**Issue by pierreozoux** [indiehosters/website#64]
Ok, we have the feeling that odoo is not right from the beginning!
I spent 3 hours to try to send the billing to users, without managing anything. It is supposedly here to help. I would have already finished if it was manual...

After trying for some months now, I would recommend to phase out this program for these 2 main reasons:
- https://github.com/odoo/odoo/issues/1839
- https://github.com/docker-library/docs/pull/435

And also, we are supporting free software, not open source.
We also want this: https://github.com/indiehosters/website/issues/49

So let's first discuss, and then phase out little by little.
We need the following:
- [ ] #66
- [x] #65 
- [ ] CRM
- [x] [forum](https://forum.indie.host/)
- [x] mass mailing (https://www.mautic.org/)
- [x] project management: wekan.indie.host
- [x] chat: chat.indie.host

I would say the most important is our payment gateway :) Once done, we can replace our `shop` link with the new one.

@pbouda any comment? Then I'll create one issue per bullet.

**Comment by pierreozoux** [comment]
And I just deleting our full newsletter contact list (without any warning...) I had to roll back a backup, first time in the existence of IndieHosters...

**Comment by pierreozoux** [comment]
I also can't save an email template. It is not even giving error nor on the client, nor on the server.

**Comment by pbouda** [comment]
Uff, I agree. For me the website editing is a pain... Let's move step-by-step, I think an issue per replacement unit is a good idea. What do you mean with payment gateway, that's the shop? CRM and shop could be two different things for now, I guess.

**Comment by pierreozoux** [comment]
Yes, CRM and shop are distinct, of course :) (I updated to reflect)

**Comment by scith** [comment]
Hi Pierre
Just for the reference, I have forked Boxbilling to make my own commits (as the devs do not seem to be very reactive): https://github.com/scith/boxbilling
As of now, I just removed the license check.
If it works well in the alpha test I will soon launch for my project (for which you are welcome of course :)), then I might develop it further. I could develop multi-lingual plugins and others if needed.

Cheers
